com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/ShaderPass/VertMesh.hlsl

```diff
@@ -136,13 +136,33 @@ VaryingsMeshType VertMesh(AttributesMesh input, float3 worldSpaceOffset)
     UNITY_SETUP_INSTANCE_ID(input);
     UNITY_TRANSFER_INSTANCE_ID(input, output);
 
+#if defined(HAVE_VFX_MODIFICATION)
+    AttributesElement element;
+    ZERO_INITIALIZE(AttributesElement, element);
+
+    // Vertex might be culled early, thus we ensure the varyings are initialized to zero.
+    ZERO_INITIALIZE(VaryingsMeshType, output);
```

Comment:
This call is redudant with the previous one above. 

 VaryingsMeshType output;
    ZERO_INITIALIZE(VaryingsMeshType, output);